### ISSUE TITLE
ns.corporation.createCorporation() - selfFund parameter is inverted

### ISSUE THREAD

**Issue by MaWo2** [danielyxie/bitburner#4115]
Hello.

As described by the title, the selfFund parameter behaves contradictory to the description.

According to the description of the `createCorporate()` command, the parameter `selfFund`, defaults to `true` with `false` being available only on BN3.
My understanding of this is, that I should set this parameter to `false`, if I want to use the seed money provided in BN3.

However, if I try this, the corporation is not created.

```
export async function main(ns) {
    var success = Boolean;
    success = ns.corporation.createCorporation("TestCorp", false);
    ns.tprint(success); //--> prints "false" to terminal
}
```

If I set the parameter to `true`, the corporation is created and seed money is used.
I suggest renaming the parameter to "seedMoney".

- [x] Minimal script
- [x] Version of the game: v2.0.2 (3067703c) (Browser version, BN3.1)

**Comment by Mughur** [comment]
this has been fixed in 2.1 already

**Comment by MaWo2** [comment]
Great. Thanks.
